paasta_tools/cleanup_tron_completed_pods.py

```diff
@@ -0,0 +1,63 @@
+import sys
+from datetime import datetime
+
+from dateutil.tz import tzutc
+
+from paasta_tools.kubernetes_tools import get_all_pods
+from paasta_tools.kubernetes_tools import get_pod_condition
+from paasta_tools.kubernetes_tools import is_pod_completed
+from paasta_tools.kubernetes_tools import KubeClient
+
+
+def main():
+    kube_client = KubeClient()
+    pods = get_all_pods(kube_client, "tron")
+    allowed_uptime_minutes = 10
```

Comment:
it's probably a good idea to make this a generic script and take in a namespace and time-to-delete as an argument

Reply:
Added